frontend/src/metabase/query_builder/components/DownloadWidget.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from "react";
+import ReactDOM from "react-dom";
+
+import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
+import Icon from "metabase/components/Icon.jsx";
+import DownloadButton from "metabase/components/DownloadButton.jsx";
+
+import FieldSet from "metabase/components/FieldSet.jsx";
+
+export default class DownloadWidget extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {};
+    }
+
+    static propTypes = {};
+    static defaultProps = {};
```

Comment:
If we aren't using `propTypes` or `defaultProps` my personal feeling is we should just omit them.